MediaBrowser.XbmcMetadata/Parsers/NfoSubtreeParsers.cs

```diff
@@ -0,0 +1,538 @@
+            int? sortOrder = null;
+            string? imageUrl = null;
+
+            reader.MoveToContent();
+            reader.Read();
+
+            // Loop through each element
+            while (!reader.EOF && reader.ReadState == ReadState.Interactive)
+            {
+                if (reader.NodeType == XmlNodeType.Element)
+                {
+                    switch (reader.Name)
+                    {
+                        case "name":
+                            name = reader.ReadStringFromNfo() ?? name;
+                            break;
+
+                        case "role":
+                            role = reader.ReadStringFromNfo() ?? role;
+                            break;
+
+                        case "type":
+                            type = NfoParserHelpers.GetPersonType(reader.ReadStringFromNfo() ?? type);
+                            break;
+
+                        case "order":
+                        case "sortorder":
+                            sortOrder = reader.ReadIntFromNfo() ?? sortOrder;
+                            break;
+
+                        case "thumb":
+                            imageUrl = reader.ReadStringFromNfo() ?? imageUrl;
+                            break;
+
+                        default:
+                            reader.Skip();
+                            break;
+                    }
+                }
+                else
+                {
+                    reader.Read();
+                }
+            }
+
+            return new PersonInfo
+            {
+                Name = name.Trim(),
+                Role = role,
+                Type = type,
+                SortOrder = sortOrder,
+                ImageUrl = imageUrl
+            };
+        }
+
+        private static void ReadRatingNode(XmlReader reader, BaseItem item, string? ratingName)
+        {
+            reader.MoveToContent();
+            reader.Read();
+
+            // Loop through each element
+            while (!reader.EOF && reader.ReadState == ReadState.Interactive)
+            {
+                if (reader.NodeType == XmlNodeType.Element)
+                {
+                    switch (reader.Name)
+                    {
+                        case "value":
+                            var val = reader.ReadElementContentAsString();
+
+                            if (!string.IsNullOrWhiteSpace(val))
+                            {
+                                if (float.TryParse(val, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture, out var ratingValue))
+                                {
+                                    // if ratingName contains tomato --> assume critic rating
+                                    if (ratingName != null &&
+                                        ratingName.Contains("tomato", StringComparison.OrdinalIgnoreCase) &&
+                                        !ratingName.Contains("audience", StringComparison.OrdinalIgnoreCase))
```

Comment:
Can you have both "tomato" and "audience" in the ratingName ?
```suggestion
                                        !ratingName.Contains("audience", StringComparison.OrdinalIgnoreCase))
```

Reply:
Yes there could be a "tomatometerallaudience" rating https://github.com/jellyfin/jellyfin/pull/4981#discussion_r554436654